packages/fether-react/src/Send/TxForm/TxDetails/TxDetails.js

```diff
@@ -8,36 +8,38 @@ import BigNumber from 'bignumber.js';
 import { fromWei, toWei } from '@parity/api/lib/util/wei';
 
 class TxDetails extends Component {
+  renderDetails = () => {
+    const { estimatedTxFee } = this.props;
+
+    return estimatedTxFee
+      ? `${this.renderCalculation()}
+${this.renderFee()}
+${this.renderTotalAmount()}`
+      : `
+Transaction incomplete..`;
```

Comment:
The reason I had it on a new line is to have it vertically centered. I'm not sure to understand why `// eslint-disable-line` could help me there.

Reply:
I thought that the commit hook that automatic fixes linting issues may have moved it onto the new line, so i was suggesting that as an option to prevent it from doing so.

I agree with your change then